src/platform/site-wide/mega-menu/containers/Main.jsx

```diff
@@ -108,6 +111,37 @@ export class Main extends Component {
     this.props.toggleMobileDisplayHidden(hidden);
   };
 
+  focusTrap = e => {
+    const buttonContainer = document.getElementById('va-nav-controls');
+    const megaMenuContainer = document.getElementById('mega-menu-mobile');
+    const focusable = [
+      ...Array.from(buttonContainer.querySelectorAll(tabbableSelectors)).filter(
```

Comment:
I think we may want some escape early checks to ensure that we're not doing `Array.from(undefined)` here 🙂

Reply:
`querySelectorAll` returns an empty array rather than `undefined`
<img width="445" alt="Screen Shot 2021-09-09 at 11 23 36 AM" src="https://user-images.githubusercontent.com/3144003/132714862-fe31e369-e689-4c91-9ffa-44a098e9304d.png">

Reply:
ahhh excellent 💯